### PR TITLE
Updated file.

### PR DESCRIPTION
**Pull request by mahesh0309**
Updated file #32 close.

### CODE REVIEW

File: tuaminx-hometown.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Hello
+
+# World
```
Comment (passion4aix) [inline tuaminx-hometown.md]: care to add something more than hello world :8ball:

**Comment by githubteacher** [comment]
@mahesh0309 your pull request is actually created backwards. So you are merging `master` into your branch instead of your branch into `master`. To fix it, close this pull request and then start a new one. In the `base` dropdown, choose `master` and in the `compare` dropdown, choose your branch!

File: tuaminx-hometown.md (hunk 1 above)
Comment (mahesh0309) [inline tuaminx-hometown.md]: @tuaminx :tongue: :laughing:

**Comment by mahesh0309** [comment]
@githubteacher thanks.

**Pull request closed without merging**